modules/core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/jdbc/dialect/Dialect.java

```diff
@@ -1015,6 +1015,20 @@ public Array createArrayOf(int type, Object[] elements, Connection connection) t
     public abstract String getUpsertSql(List<Column> columns, List<Serializable> values, List<Column> outColumns,
             List<Serializable> outValues);
 
+    /**
+     * Gets the SQL string for an INSERT ... ON CONFLICT DO NOTHING expression.
```

Comment:
on conflict of what? doing nothing means returns null SQL or skip the column or override?